extension/content_script.js

```diff
@@ -263,6 +268,7 @@ function unhighlightEverything() {
 	if (tracker.isTracking()) {
 		let sentenceId = tracker.getSentenceId();
 		doc.getSentenceEls(tracker.getSentenceId()).removeClass(trackerStyle.getSentenceStyle());
+		doc.getSentenceKeywordsEls(tracker.getSentenceId()).removeClass(trackerStyle.getKeywordStyle());
 	}
 	$("." + keywordClass).unmark();
```

Comment:
We probably don't need this anymore?
-- Also maybe ctrl - F all usages of keywordClass and make sure they all use the preprocessed keywordEls instead.

Reply:
👍